big_tests/tests/service_mongoose_system_stats_SUITE.erl

```diff
@@ -59,34 +60,42 @@ init_per_group(_GroupName, Config) ->
 end_per_group(_GroupName, Config) ->
     Config.
 
-init_per_testcase(user_stats_are_reported_to_google_analytics_when_mim_starts, Config) ->
+init_per_testcase(system_stats_are_reported_to_google_analytics_when_mim_starts, Config) ->
     create_events_collection(),
-    enable_user_stats(mim()),
+    enable_system_stats(mim()),
     delete_prev_client_id(mim()),
     Config;
+init_per_testcase(system_stats_are_not_reported_when_not_allowed, Config) ->
+    create_events_collection(),
+    disable_system_stats(mim3()),
```

Comment:
Again, what do we need mim3 for? Not that I have anything against, on the opposite, this might be good to test elsewhere, poor mim1 already has too many things to do. Maybe we could even make these tests run parallel if they don't interfere with each other, but in any case it's good to leave comments or so 🙂

Reply:
Using mim3 was a debris form previous implementation. Changed the test to be using mim for the test when gathering the stats is not allowed.